coalib/bearlib/aspects/Smell.py

```diff
@@ -75,21 +74,19 @@ class docs:
 @Smell.subaspect
 class MethodSmell:
     """
-    This aspect detects `code smells` related to methods' and functions
-    definitions in your codebase.
+    Code smells related to a method or function definition.
 
     Method-level code smells indicate poorly defined method and or
     functions (too long method or functions, or functions with too many
     parameters) in your source code.
     """
     class docs:
         example = """
-        # This function has way too many parameters
-
-        def func(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, z):
-            pass
+        * Long method
+        * Long parameter list
+        etc.
         """
-        example_language = 'python'
+        example_language = 'English'
```

Comment:
But `def func(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, z):` is Python code right?